### PR TITLE
Libosinfo (Part I)

### PR DESCRIPTION
**Pull request by fidencio**
First part of the blog post series about libosinfo.

### CODE REVIEW

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+
+### A quick overview
+
+Libosinfo is **the** operating system information database. As a project, it
+consists of three different parts, with the goal to provide a single place
+containing all the information about an operating system that is required in
+order to provision and manage it in a virtualized environment.
+
+The project allows management application to:
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: 'a management application' or 'applications'

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+* [GNOME Boxes] (https://wiki.gnome.org/Apps/Boxes): A simple application to
+  view, access, and manage remote and virtual systems.
+
+* [Cockpit Machines]
+  (https://cockpit-project/guide/latest/feature-virtualmachines): A [Cockpit]
+  (https://cockpit-project.org/) extension to manage virtual machines running
+  on the host.
+
+#### ... and for what they use it
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: "what they use it for" sounds better to my ears

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  system's vendors, management applications can rely on that when setting
+  the default resources for an installation.
+
+    - [x] OpenStack Nova
+    - [x] virt-manager
+    - [x] GNOME Boxes
+    - [x] Cockpit Machines
+
+* **Download ISOs**: As libosinfo provides the ISOs URLs, management
+  applications can offer the user the option to download a specific a specific
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: duplicate "a specific"

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+  operating system;
+
+    - [ ] OpenStack Nova
+    - [ ] virt-manager
+    - [x] GNOME Boxes
+    - [ ] Cockpit Machines
+
+* **Start tree installations**: As libosinfo provides the tree installations
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: installation

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+  installation without having to download the whole operating system ISO;
+
+    - [ ] OpenStack Nova
+    - [x] virt-manager (via virt-install)
+    - [ ] GNOME Boxes
+    - [ ] Cockpit Machines
+
+* **Automatically set the hardware supported**: As libosinfo provides the list
+  of hardware supported by an operating system, management applications can
+  choose which hardware to use either base in this information;
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: something is missing in this phrase.

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+    - [x] OpenStack Nova
+    - [x] virt-manager
+    - [x] GNOME Boxes
+    - [x] Cockpit Machines (via virt-install)
+
+
+* **Unattend install**: as libosinfo provides unattended installations scripts
```
Comment (r4f4) [inline content/posts/2019-10-16-libosinfo-part-I.md]: Unattended install

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+  view, access, and manage remote and virtual systems.
+
+* [Cockpit Machines]
+  (https://cockpit-project/guide/latest/feature-virtualmachines): A [Cockpit]
+  (https://cockpit-project.org/) extension to manage virtual machines running
+  on the host.
+
+#### ... and for what they use it
+
+* **Download ISOs**: As libosinfo provides the ISOs URLs, management
```
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
* **Download ISOs**: As libosinfo provides the ISO URLs, management
```

File: content/posts/2019-10-16-libosinfo-part-I.md (hunk 2 above)
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
#### ... and why they use it
```

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+  operating system of an ISO, managament applications can use this info to
+  set reasonable default values for resources, to select the hardware
+  supported, and to perform unattended installations.
+
+    - [ ] OpenStack Nova
+    - [x] virt-manager
+    - [x] GNOME Boxes
+    - [x] Cockpit Machines
+
+* **Start tree installations**: As libosinfo provides the tree installations
```
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
* **Start tree installations**: As libosinfo provides the tree installation
```

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+  URLs, management applications can use it to start a network-based
+  installation without having to download the whole operating system ISO;
+
+    - [ ] OpenStack Nova
+    - [x] virt-manager (via virt-install)
+    - [ ] GNOME Boxes
+    - [x] Cockpit Machines
+
+* **Set reasonable default values for RAM, CPU, and disk resources**: As
+  libosinfo has the values, which are the ones recommended by the operating
```
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
  libosinfo knows the values that are recommended by the operating
```

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+  the default resources for an installation.
+
+    - [x] OpenStack Nova
+    - [x] virt-manager
+    - [x] GNOME Boxes
+    - [x] Cockpit Machines
+
+* **Automatically set the hardware supported**: As libosinfo provides the list
+  of hardware supported by an operating system, management applications can
+  choose which hardware to use either base in this information;
```
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
  choose which hardware to use based on this information;
```

File: content/posts/2019-10-16-libosinfo-part-I.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+
+    - [x] OpenStack Nova
+    - [x] virt-manager
+    - [x] GNOME Boxes
+    - [x] Cockpit Machines (via virt-install)
+
+
+* **Unattend install**: as libosinfo provides unattended installations scripts
+  for CentOS, Debian, Fedora, Fedora Silverblue, Microsoft Windows, OpenSUSE,
+  Red Hat Enterprise Linux, and Ubuntu, managament applications can perform
```
Comment (jonner) [inline content/posts/2019-10-16-libosinfo-part-I.md]:
```suggestion
  Red Hat Enterprise Linux, and Ubuntu, management applications can perform
```

**Review by jonner — COMMENTED** [review]
Quite clear now. Just a few minor suggestions.

**Pull request merged**